### ISSUE TITLE
Disable warnings?

### ISSUE THREAD

**Issue by embray** [astropy/astropy4scipy2014#12]
I've noticed that a few of the samples display copious warning messages.  To make matters worse, on IPython 2.0 at least, the notebook shows a duplicate copy of each warning message.  It's quite noisy.

On the one hand, some of the warnings are helpful to see--when users go off on their own they'll know that those are supposed to be there, and some of them are informative.  On other cases I think they're far too noisy.

Should we consider disabling warnings at the beginning of each notebook?  Or at least show how to do it?

**Comment by astrofrog** [comment]
I agree it's a bit noisy. It's a pain the warnings are duplicated too. Maybe we can explain where the warnings come from and show how to disable them?
